db/migrate/20150313054707_create_translations.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTranslations < ActiveRecord::Migration
+  def change
+    create_table :translations do |t|
+      t.datetime :time_stamp
```

Comment:
This timestamp variable is ambiguous. Rename it to something more clear, such as "date_assigned."